Remove deprecated plain text auth references for wodles (#6405)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/wodle-azure-logs.rst

Reviewer: Only examples should be removed for now. If you prefer, you can open a new issue for it and assign this PR to the new issue. Then, create a PR for the original issue (https://github.com/wazuh/wazuh-documentation/issues/6363) only for removing examples.

This block configures the integration with Azure Log Analytics REST API.

- `log_analytics\\auth_path`_
- `log_analytics\\tenantdomain`_
- `log_analytics\\request`_

+----------------------------------------+----------------------------------------------+
| Options                                | Allowed values                               |
+========================================+==============================================+
| `log_analytics\\auth_path`_            | File path                                    |
+----------------------------------------+----------------------------------------------+
| `log_analytics\\tenantdomain`_         | Any string                                   |
+----------------------------------------+----------------------------------------------+
| `log_analytics\\request`_              | N/A                                          |
+----------------------------------------+----------------------------------------------+

log_analytics\\auth_path
^^^^^^^^^^^^^^^^^^^^^^^^

> Path of the file that contains the application identifier and the application key for authentication in order to use the Azure Log Analytics API.

Reviewer 2: ```suggestion
File path containing the application identifier and key for authentication. This is necessary to use the Azure Log Analytics API.
```

This block configures the integration with Azure Active Directory Graph REST API.

- `graph\\auth_path`_
- `graph\\tenantdomain`_
- `graph\\request`_

+----------------------------------+----------------------------------------------+
| Options                          | Allowed values                               |
+==================================+==============================================+
| `graph\\auth_path`_              | File path                                    |
+----------------------------------+----------------------------------------------+
| `graph\\tenantdomain`_           | Any string                                   |
+----------------------------------+----------------------------------------------+
| `graph\\request`_                | N/A                                          |
+----------------------------------+----------------------------------------------+

graph\\auth_path
^^^^^^^^^^^^^^^^

> Path of the file that contains the application identifier and the application key for authentication in order to use the AAD Graph API.

Reviewer 2: ```suggestion
File path containing the application identifier and key for authentication. This is necessary to use the AAD Graph API.
```

This block configures the integration with Azure Storage.

- `storage\\auth_path`_
- `storage\\tag`_
- `storage\\container`_

+----------------------------------+----------------------------------------------+
| Options                          | Allowed values                               |
+==================================+==============================================+
| `storage\\auth_path`_            | File path                                    |
+----------------------------------+----------------------------------------------+
| `storage\\tag`_                  | Any string                                   |
+----------------------------------+----------------------------------------------+
| `storage\\container`_            | N/A                                          |
+----------------------------------+----------------------------------------------+

storage\\auth_path
^^^^^^^^^^^^^^^^^^

> Path of the file that contains the account name and the account key for authentication.

Reviewer 2: ```suggestion
File path containing the account name and key for authentication.
```
